pkg/codegen/python/gen.go

```diff
@@ -367,6 +367,10 @@ func tokenToModule(tok string, pkg schema.PackageReference, moduleNameOverrides
 		pkg = (&schema.Package{}).Reference()
 	}
 	canonicalModName := pkg.TokenToModule(tok)
+	return moduleToPythonModule(canonicalModName, moduleNameOverrides)
+}
+
+func moduleToPythonModule(canonicalModName string, moduleNameOverrides map[string]string) string {
```

Comment:
Again docs and examples would be awesome here. I know it's not necessarily new functionality you've added, but I imagine as part of this you have a picture of what goes into/comes out of this function, and a couple of links capturing that would be ace!